### PR TITLE
Added VaTable (new Table Component) with props headers, items and list.

### PR DESCRIPTION
**Pull request by guastallaigor**

### CODE REVIEW

File: src/components/Table.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template lang="html">
+  <table class="aui" :class="list ? 'aui-table-list' : ''">
+    <thead>
```
[the remark this answers is not in the document]
Comment (guastallaigor) [reply src/components/Table.vue]: Thank you for the suggestion. 

I added one default slot for `<tbody>` and `<thead>` (if the user wants) and also added a slot with the name footer, for the `<tfoot>`, what do you think? 

So, now works with the slots **or** the headers/items prop, **not** both.
I also added a `v-if` that don't render the `<tbody>` and `<thead>` when there's no items.length. 

But the props are now **not required**.

File: src/components/Table.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <tr>
+        <th v-for="(header, keyH) in headers" :key="keyH" :id="keyH">{{ header }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(item, keyV) in items" :key="keyV">
+        <td v-for="(value, keyI) in Object.values(item)" :key="keyI" :headers="keyI">{{ value }}</td>
```
[the remark this answers is not in the document]
Comment (guastallaigor) [reply src/components/Table.vue]: I'm not sure yet, but when I get some more time I will try them. Thank you for the fast replies.

**Comment by guastallaigor** [comment]
Yes, I just did. I added this _"2. Specify object key to look for in the header itself"_ that was missing and now it can be reorder based on the `key` property on any object inside the headers array.

Please, check it out and if you want any more changes just let me know.

**Comment by damienix** [comment]
The functionality looks good now. Just the description in the documentation is outdated now...

**Pull request merged**